tables_io/io_layer.py

```diff
@@ -88,6 +93,46 @@ def tableToDataframe(tab):
     return df
 
 
+def dataframeToArrayDict(df):
```

Comment:
Maybe (or TODO later) add an optional slice argument to these that is passed through?

Reply:
Could do, but I was thinking that rather re-implement functionality that is available in dataframe already, why not just expect the user to do all the slicing and dicing on the dataframe itself, before calling this method?

Thoughts?

Reply:
I guess the exception is if this function is called when reading the dataframe.

Reply:
Think I put this comment in the wrong place and was talking about the IO things below.